gstreamer-video/src/video_info.rs

```diff
@@ -190,6 +190,14 @@ impl ::VideoMultiviewFramePacking {
 
 pub struct VideoInfo(pub(crate) ffi::GstVideoInfo);
 
+impl fmt::Debug for VideoInfo {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{:?} - ({:?}x{:?} - {:?}bytes)", self.format(),
+            self.width(), self.height(), self.size())
```

Comment:
This should use the `debug_struct` API like the others

Reply:
Done.